### PR TITLE
Batch 2: SEO H1 headings

### PR DESCRIPTION
**Pull request by lhcole613**
FREELANCERS: Please do NOT merge as per @pebaum.

### CODE REVIEW

File: microsoft-365/admin/misc/device-list.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ ROBOTS: NOINDEX
 description: "Learn how to make a CSV file for AutoPilot in Microsoft 365 for business."
 ---
 
-# Device list CSV-file
+# Windows Autopilot device list CSV-file
```
Comment (SKjerland) [inline microsoft-365/admin/misc/device-list.md:28]: @lhcole613 - Shouldn't we include the product name for SEO? "Windows Autopilot device list CSV-file in Microsoft 365 for business"

File: microsoft-365/admin/misc/icann-verification-of-contact-information.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ ROBOTS: NOINDEX
 description: "Learn how to validate your contact information to comply with ICANN regulations."
 ---
 
-# ICANN Verification of Contact Information
+# ICANN verification of contact information
```
Comment (SKjerland) [inline microsoft-365/admin/misc/icann-verification-of-contact-information.md:25]: @lhcole613 - Same comment as above about adding product name. Should this be "ICANN verification of contact information in Microsoft 365 for business" or whichever product this is in (I'm not sure from this context)?

File: microsoft-365/admin/misc/power-bi-in-your-organization.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ ROBOTS: NOINDEX
 description: "Learn about Power BI and how users in your organization can use this business analytics service."
 ---
 
-# Power BI in your organization
+# Using Power BI data in your organization
```
Comment (SKjerland) [inline microsoft-365/admin/misc/power-bi-in-your-organization.md:24]: @lhcole613 - For SEO, shouldn't the product name be mentioned in this H1?

File: microsoft-365/admin/productivity/communication.md [hunk 4]
```diff
@@ -21,9 +21,9 @@ search.appverid:
 description: "Details of the Communication - People experience Productivity Score."
 ---
 
-# Communication – People experiences
+# Communication insights score– People experiences
```
Comment (SKjerland) [inline microsoft-365/admin/productivity/communication.md:24]: @lhcole613 - Add product name to H1 for SEO?

File: microsoft-365/admin/productivity/meetings.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ search.appverid:
 description: "Details of the Meetings - people experiences Productivity score."
 ---
 
-# Meetings – People experiences
+# Meetings insights score – People experiences
```
Comment (SKjerland) [inline microsoft-365/admin/productivity/meetings.md:24]: @lhcole613 - Add product name to H1 for SEO?

File: microsoft-365/admin/productivity/mobility.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ search.appverid:
 description: "Details of the Mobility - people experiences Productivity score."
 ---
 
-# Mobility – People experiences
+# Mobility insights score – People experiences
```
Comment (SKjerland) [inline microsoft-365/admin/productivity/mobility.md:24]: @lhcole613 - Add product name to H1 for SEO?

File: microsoft-365/admin/misc/power-bi-in-your-organization.md (hunk 3 above)
Comment (SKjerland) [inline microsoft-365/admin/misc/power-bi-in-your-organization.md:24]: @lhcole613 - I think you'll get dinged by a PubOps reviewer if you use a gerund in the heading, unless they changed this rule. They forced me to change a gerund once. Maybe change to "Overview of Power BI in your organization" if it's an issue.

File: microsoft-365/business-premium/m365-campaigns-users.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ search.appverid:
 description: "Learn how these security recommendations for Microsoft 365 Business Premium affects your users and protects your data."
 ---
```
Comment (SKjerland) [inline microsoft-365/business-premium/m365-campaigns-users.md:27]: @lhcole613 - Can you please change "affects" to "affect" and "protects" to "protect" in the metadata description? Thanks!

**Review by SKjerland — CHANGES_REQUESTED** [review]
@lhcole613 - Please see my individual comments inline. Thanks!

**Comment by officedocspr2** [comment]
## STOP  Caution: Large pull request

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **32** files, which exceeds the **warning** limit of **32**.

Before this pull request can be merged, you must review the list of files it contains and their changes to confirm they are correct and intentional. **This includes files you did not change and/or do not own**. If there are changes you aren't familiar with, contact the individuals who submitted the commits included in this pull request and review the changes with them.

**The individual who merges this pull request is responsible for ensuring the changes included in it are correct, and for resolving any issues that might result in merging it.**

### To merge this pull request

If you have determined that the changes in this pull request are correct, add the **Merge large pull request**<sup>\*</sup> label to it, wait for the **max/pr-file-count** check to pass, and then click **Merge**. To add a label, click the gear icon next to **Labels** and then select **Merge large pull request**<sup>\*</sup>.

Label UI screenshot

### To reject this pull request

If you can't confirm the changes in this pull request are correct, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

**Pull request merged**